tools/memleak.py

```diff
@@ -206,6 +236,19 @@ def run_command_get_pid(command):
 bpf_source = open("memleak.c").read()
```

Comment:
I didn't catch this the first time around, but this will only work if the tool is run from the tools source directory. See the _find_files() function of BPF for a possible helper you could use. We need a utils submodule it seems...